sssd_test_framework/utils/sssd.py

```diff
@@ -783,3 +784,42 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def nslcd(
+        self,
+        provider: str | list[str] = "id",
+        role: Optional[MultihostRole] = None,
+        domain: str | None = None,
+    ):
+        """
+        :param provider: SSSD providers (id, auth, chpass, ...)
+        :type provider: str | list[str]
+        :param role: MultihostRole (LDAP server)
+        :type role: MultihostRole
+        :param domain: Existing domain name, defaults to None (= default domain)
+        :type domain: str | None, optional
+        """
+        if domain is None:
+            domain = self.sssd.default_domain
+
+        self.sssd.fs.write(
+            "/etc/pam.d/sssdproxyldap",
+            """
+                auth    required pam_ldap.so
+                account required pam_ldap.so
+                password required pam_ldap.so
+                session required pam_ldap.so
+                """,
+        )
+
+        self.sssd.fs.write(
+            "/etc/nslcd.conf", f"uid nslcd\ngid ldap\nuri " f"ldap://{role.host.hostname}\n", dedent=False
```

Comment:
its black formatted

Reply:
The other way can also be black formatted as well. Let's please, adhere to how this file is written.

Reply:
Please provide a better reason, then it's just "it's black formatted"

`f"uid nslcd\ngid ldap\nuri\n ldap://{role.host.hostname}\n"`

I'm sure this will pass the checks as well. It's cleaner, it matches the rest of the file. Why do we even need 

The f if now variable is not specified? It could also be "uid nslcd\nbid ldap\nuri " f"ldap://{role.host.hostname}\n"

Reply:
I already changed this to :  f"uid nslcd\ngid ldap\nuri ldap://{role.host.hostname}\n"